src/client/components/PlayList/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./styles.css";
+
+import { PlayListSong } from "./components/PlayListSong";
+import { PlayListTitle } from "../PlayListTitle";
+
+
+
+export function PlayList(props) {
+  return (
+      <div className="playlist">
+        <PlayListTitle />
+        <div className="playlist-filter">
+          <div className="playlist-filter-hot" onClick={()=>props.onUpdateFilter("hot")}>H O T</div>
```

Comment:
the classes playlist-filter-hot and playlist-filter-new contain the same css rules. Instead of having two classes, just use the same for both divs